proto/autokitteh/remote/v1/runner_manager_svc.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package autokitteh.remote.v1;
+
+import "autokitteh/remote/v1/types.proto";
+
+message ContainerConfig {
+  string image = 1;
+  // TBD by @efiShtain
+}
+
+// TODO: Will become Start once we split to files
+// Tell runner manager to start a runner
+message StartRunnerRequest {
```

Comment:
If we're going the splitting path, let have sub directories per service (runner, runner_manager, ...) This will simplify naming. For example this will be `StartRequest`.

Reply:
I'll see if I can make it work

Reply:
done

could you please re review ?